### ISSUE TITLE
Alternative to Tabs outline

### ISSUE THREAD

**Issue by Abbe98** [wikimaps-dev/wikimaps-html#8]
When selecting a tab it creates the default ugly outline. we could replace this outline with some thing both better for accessibility and looks.

**Comment by susannaanas** [comment]
Agree. What do you have in mind for accessibility?

**Comment by Abbe98** [comment]
A Mediawiki blue 3px border at the top or bottom of the menu item.

**Comment by susannaanas** [comment]
I would be happy to remove it. I only managed to turn it into black. It is a property of the browser, I read. Here's one link http://stackoverflow.com/questions/13838377/jquery-ui-remove-orange-tab-border-on-focus

**Comment by Abbe98** [comment]
Removing it is just `outline: none;` ;-)
